Asset tracker updates (#381)
NordicPlayground/fw-nrfconnect-nrf: samples/nrf9160/asset_tracker/README.rst

nRF9160: Asset Tracker
######################

The Asset Tracker sample demonstrates how to use the :ref:`lib_nrf_cloud` to connect an nRF9160 DK to the `nRF Cloud`_ via LTE and transmit GPS and sensor data.

Overview
********

> The sample sends GPS position data, temperature and accelerometer data that is collected by an nRF9160 DK to Nordic Semiconductor's cloud solution, `nRF Cloud`_, where the data is visualized.

Reviewer: ```suggestion
The sample sends GPS position, temperature, and accelerometer data that is collected by an nRF9160 DK to Nordic Semiconductor's cloud solution, `nRF Cloud`_, where the data is visualized.
```

The accelerometer data is used to infer the device's physical orientation, which is sent as the nRF Cloud sensor data type "FLIP".
Temperature data is sent as the nRF Cloud sensor type "TEMP".

By default, the sample uses simulated sensor and GPS data, but it can be configured with Kconfig options to use real sensors to collect data.
The LTE link control driver is used to establish the LTE link automatically.

> When the device is connected to nRF Cloud, the nRF 9x Link Monitor can be used to send AT commands and receive the response.

Reviewer: ```suggestion
When the device is connected to the nRF Cloud, you can use the LTE Link Monitor to send AT commands and receive the response.
```

It can be programmed independently from the secure boot firmware.

See :ref:`gs_programming` for information about how to build and program the application.
The Kconfig file of the sample contains options to configure the sample.

> For example, configure :option:`CONFIG_POWER_OPTIMIZATION_ENABLE` to enable power optimization or :option:`CONFIG_TEMP_USE_EXTERNAL` to use an external temperature sensor instead of simulated temperature data. 

Reviewer: ```suggestion
For example, configure ``CONFIG_POWER_OPTIMIZATION_ENABLE`` to enable power optimization or ``CONFIG_TEMP_USE_EXTERNAL`` to use an external temperature sensor instead of simulated temperature data.
```

Reviewer: Sorry, messed up here - we can only link to config options for libraries etc., not for samples.

It can be programmed independently from the secure boot firmware.

See :ref:`gs_programming` for information about how to build and program the application.

> The sample has a Kconfig file with options to configure the sample, such as enabling power optimization and selecting simulated or real sensor data. These can be browsed and configured using the command line tool ``menuconfig`` or directly in ``prj.conf``.

Reviewer: ```suggestion
The Kconfig file of the sample contains options to configure the sample.
For example, configure :option:`CONFIG_POWER_OPTIMIZATION_ENABLE` to enable power optimization or :option:`CONFIG_TEMP_USE_EXTERNAL` to use an external temperature sensor instead of simulated temperature data. 
In |SES|, select **Project** > **Configure nRF Connect SDK project** to browse and configure these options.
Alternatively, use the command line tool ``menuconfig`` or configure the options directly in ``prj.conf``.
```

1. Connect the board to the computer using a USB cable.
   The board is assigned a COM port (Windows) or ttyACM device (Linux), which is visible in the Device Manager.

> #. Connect to the board with a terminal emulator, for example nRF9x Link Monitor.

Reviewer: ```suggestion
#. Connect to the board with a terminal emulator, for example, LTE Link Monitor.
```
